test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -5590,6 +5600,38 @@ def forward(self, x):
         x = torch.randn(2, 3, 4)
         self.run_test(SiLUModel(), (x))
 
+    @skipIfUnsupportedMinOpsetVersion(14)
+    def test_tril(self):
+        class trilModel(torch.nn.Module):
+            def forward(self, x):
+                return torch.tril(x)
+
+        x = torch.randn(2, 3, 4)
+        self.run_test(trilModel(), (x))
+
+        class trilModelwithDiagonal(torch.nn.Module):
```

Comment:
Can we add a new test for negative value of Diagonal, like -1? The logic of this is different than positive one.

Reply:
Added :)